src/com/epam/course/hometask3/Authors.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.course.hometask3;
+
+class Authors extends Book {
```

Comment:
Extending Author from Book doesn't make much sense. These should be separate entities in terms of inheritance.